# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: devtools/server/actors/watcher/target-helpers/moz.build
# Commit: 6eaa4d12b58c
# Full Hash: 6eaa4d12b58cb4762729fdf53511450926383db8
# Author: Nicolas Chevobbe <nchevobbe@mozilla.com>
# Date: 2020-10-15 09:40:06
# Regressor Bug: 1633712
# File Overlap Count: 1
# Description:
#   Bug 1633712 - [devtools] Create WorkerTargets as soon as possible. r=ochameau,devtools-backward-compat-reviewers.
#   
#   This patch adds support for dedicated worker targets in the Watcher actor.
#   Shared and Service workers are not handled yet.
#   
# ==============================================================================

diff -r 10f97f47c883 -r 6eaa4d12b58c devtools/server/actors/watcher/target-helpers/moz.build
--- a/devtools/server/actors/watcher/target-helpers/moz.build	Thu Oct 15 05:11:58 2020 +0000
+++ b/devtools/server/actors/watcher/target-helpers/moz.build	Thu Oct 15 05:17:52 2020 +0000
@@ -7,5 +7,6 @@
 DevToolsModules(
     'frame-helper.js',
     'utils.js',
+    'worker-helper.js',
 )
 